src/Header/SetCookie.php

```diff
@@ -486,18 +492,18 @@ public function setQuoteFieldValue($quotedValue)
     }
 
     /**
-     * @return bool
+     * @return null|bool
      */
     public function isSecure()
     {
         return $this->secure;
     }
 
     /**
-     * @param  bool $httponly
+     * @param  null|bool $httponly
      * @return SetCookie
      */
-    public function setHttponly($httponly)
+    public function setHttpOnly($httponly)
```

Comment:
This is a breaking change; please revert, and also revert any calls to the method (I saw several above, but didn't realize it was because you'd renamed the method).

Alternatively, keep the original, and have it proxy to a new method named `setHttpOnly()` — though I'm not sure if PHP will allow it.

Reply:
Reverted. Methods are case insensitive.